.github/workflows/continuous-integration.yml

```diff
@@ -820,28 +807,58 @@ jobs:
 
       - name: Upload build
         run: aws s3 cp ${{ matrix.buildtype }}.tar.bz2 s3://vetsgov-website-builds-s3-upload/application-build/$GITHUB_SHA/${{ matrix.buildtype }}.tar.bz2 --acl public-read --region us-gov-west-1
+      
+  get-latest-run-number:
+    name: Get Latest Workflow Run Number
+    runs-on: ubuntu-latest
+    if: ${{ always() && github.ref == 'refs/heads/master' && needs.archive.result == 'success' }}
+    needs: archive
+    outputs:
+      latest_run_number: ${{ steps.latest-run-number.outputs.latest_run_number }}
 
-      - name: Test Current SHA
-        run: echo ${{ github.sha }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
 
-      - name: Test Current SHA with REVPARSE
-        run: git rev-parse HEAD
-        continue-on-error: true
+      - name: Get Node version
+        id: get-node-version
+        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
 
-      - name: Test Origin SHA with REVPARSE
-        run: git rev-parse origin/master
-        continue-on-error: true
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ steps.get-node-version.outputs.NODE_VERSION }}
 
-      - name: Set output of origin_master_sha
-        id: origin_master_sha
-        run: echo ::set-output name=origin_master_sha::$(git rev-parse origin/master)
-        continue-on-error: true
+      - name: Cache dependencies
+        id: cache-dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            .cache/yarn
+            node_modules
+          key: ${{ steps.get-node-version.outputs.NODE_VERSION }}-${{ hashFiles('yarn.lock') }}
+          restore-keys: ${{ steps.get-node-version.outputs.NODE_VERSION }}-
```

Comment:
Lets comment this out since there is some funky stuff going on between GHA & Jenkins